pages/about.html

```diff
@@ -1,12 +1,105 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/static/styles.css" />
+
+    <title>About Yonese</title>
+    
 </head>
 <body>
-  
+
+<!-- Navbar Section -->
+<nav class="navbar">
+    <div class="navbar__container">
+        <a href="index.html" id="navbar__logo">Yonese James Portfolio</a>
+        <div class="navbar__toggle" id="mobile-menu">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        <ul class="navbar__menu">
+            <li class="navbar__item">
+                <a href="index.html" class="navbar__links">Home</a>
+            </li>
+            <li class="navbar__item">
+              <a href="about.html" class="navbar__links">About Yonese</a>
+            </li>
+            <li class="navbar__item">
+                <a href="portfolio.html" class="navbar__links">Portfolio</a>
+            </li>
+            <li class="navbar__btn">
+                <a href="" class="button">Resume</a>
+            </li>
+            <li class="navbar__btn">
+                <a href="" class="button">Contact Me</a>
+            </li>
+        </ul>
+    </div>
+</nav>
+
+
+<div class="main">
+  <div class="main__container">
+      <div class="main__content">
+          <h1>Yonese James</h1>
+          <p>Your friendly neighborhood programmer!</p>
+      </div>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Blandit cursus risus at ultrices mi tempus imperdiet nulla malesuada. Sit amet consectetur adipiscing elit pellentesque habitant morbi tristique senectus. Integer vitae justo eget magna. Eros donec ac odio tempor. Ut faucibus pulvinar elementum integer enim. Euismod quis viverra nibh cras pulvinar. Pellentesque sit amet porttitor eget dolor morbi non. Massa tempor nec feugiat nisl pretium fusce id. Ornare quam viverra orci sagittis. Sem fringilla ut morbi tincidunt augue interdum velit euismod in. Imperdiet massa tincidunt nunc pulvinar sapien et ligula ullamcorper.</p>
+  </div>
+</div>
+
+
+<!-- Footer Section -->
+<div class="footer__container">
+    <div class="footer__links">
+        <div class="footer__link--wrapper">
+            <div class="footer__link--items">
+                <h2>About</h2>
+                <a href="">About Me</a>
+            </div>
+            <div class="footer__link--items">
+                <h2>Projects</h2>
+                <a href="">Project 1</a>
+                <a href="">Project 2</a>
+                <a href="">Project 3</a>
+            </div>
+        </div>
+        <div class="footer__link--wrapper">
+            <div class="footer__link--items">
+                <h2>Contact</h2>
+                <a href="">Contact Me</a>
+                <a href="">Resume</a>
+            </div>
```

Comment:
Since these are all clickable links, you could put these in a nav element nested in the footer element.

```HTML
<footer>
    <nav>
    <section class="about-me">
    <section class="projects">
    <section class="contact-me">
```

Then within each section you could have what you needed to make the content semantically meaningful. For example, the section element for projects could have a nested unordered list and each link to your individual project would be a list item